### PR TITLE
docs: Adding example for FieldValue.isEqual

### PR DESCRIPTION
**Pull request by schmidt-sebastian**
Fixes b/129393978

### CODE REVIEW

File: dev/src/field-value.ts [hunk 1]
```diff
@@ -171,6 +171,28 @@ export class FieldValue {
    *
    * @param {*} other The value to compare against.
    * @return {boolean} true if this `FieldValue` is equal to the provided value.
+   *
+   * @example
+   * let fieldValues = [
```
Comment (mikelehen) [inline dev/src/field-value.ts]: const?

Comment (mikelehen) [reply dev/src/field-value.ts]: (very optional)

Comment (schmidt-sebastian) [reply dev/src/field-value.ts]: For better or for worse, we don't use `const` in any of the other samples. Leaving as is.

Comment (schmidt-sebastian) [reply dev/src/field-value.ts]: For better or for worse, we don't use `const` in any of the other samples. Leaving as is.

**Pull request merged**